### PR TITLE
Remove big endian subsystems

### PR DESCRIPTION
**Pull request by dawagner**
Unused feature.
<a href='#crh-start'></a><a href='#crh-data-%7B%22processed%22%3A%20%5B%22https%3A//github.com/01org/parameter-framework/pull/265%23issuecomment-144971719%22%2C%20%22https%3A//github.com/01org/parameter-framework/pull/265%23discussion_r41007137%22%2C%20%22https%3A//github.com/01org/parameter-framework/pull/265%23discussion_r41014285%22%5D%2C%20%22comments%22%3A%20%7B%22General%20Comment%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/01org/parameter-framework/pull/265%23issuecomment-144971719%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22%40krocard%20please%20review.%22%2C%20%22created_at%22%3A%20%222015-10-02T09%3A23%3A44Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/6430928%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/dawagner%22%7D%7D%5D%2C%20%22title%22%3A%20%22General%20Comment%22%7D%2C%20%22Pull%209dd781239e041872b70e9d34ecb69a3b39b079b4%20doc/requirements/requirements.md%2010%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/01org/parameter-framework/pull/265%23discussion_r41007137%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22I%20would%20rather%20put%20that%20in%20the%20next%20section%20%28even%20next/long%20term%29.%22%2C%20%22created_at%22%3A%20%222015-10-02T10%3A02%3A23Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/6862950%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/krocard%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3A%2B1%3A%20done%22%2C%20%22created_at%22%3A%20%222015-10-02T12%3A00%3A26Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/6430928%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/dawagner%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20doc/requirements/requirements.md%3AL333-346%22%7D%7D%7D'></a>
<a href='https://www.codereviewhub.com/'></a>
- [ ] <a href='#crh-comment-General Comment'></a> &nbsp;<b><a href='https://github.com/01org/parameter-framework/pull/265#issuecomment-144971719'>General Comment</a></b>
- <a href='https://github.com/dawagner'></a> @krocard please review.
- [x] <a href='#crh-comment-Pull 9dd781239e041872b70e9d34ecb69a3b39b079b4 doc/requirements/requirements.md 10'></a> &nbsp;<b><a href='https://github.com/01org/parameter-framework/pull/265#discussion_r41007137'>File: doc/requirements/requirements.md:L333-346</a></b>
- <a href='https://github.com/krocard'></a> I would rather put that in the next section (even next/long term).
- <a href='https://github.com/dawagner'></a> :+1: done

<a href='https://www.codereviewhub.com/01org/parameter-framework/pull/265?mark_as_completed=1'></a>&nbsp;<a href='https://www.codereviewhub.com/01org/parameter-framework/pull/265?approve=1'></a>&nbsp;<a href='https://github.com/01org/parameter-framework/pull/265'></a>
<a href='#crh-end'></a>

### CODE REVIEW

**Comment by dawagner** [comment]
This pull request is now managed with CodeReviewHub. <a href='https://github.com/01org/parameter-framework/pull/265'>Refresh</a>.<a href='#crh-update-none'></a>

**Comment by dawagner** [comment]
@krocard please review.

File: doc/requirements/requirements.md [hunk 1]
```diff
@@ -333,6 +333,13 @@ The following parameter adaptation **SHOULD** be supported
    <note>The reference implementation also supports passing a floor value to be
    applied after conversion.</note>
 
+ - Endianness adaptation: A parameter or a block of parameters might be
+   presented by the Parameter Framework but only used as a passthrough to the
+   underlying subsystem (think "(void *) interfaces"). It is then possible that
+   the endianess of the subsystem differs from the one the Parameter Framework
+   is running on, an endianness adaptation allows to support those cases.
+   <ko>Not implemented in the reference implementation.</ko>
+
```
Comment (krocard) [inline doc/requirements/requirements.md]: I would rather put that in the next section (even next/long term).

Comment (dawagner) [inline doc/requirements/requirements.md]: :+1: done

**Pull request merged**